docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,211 @@
+prior to installation.
+</Admonition>
+
+### Group import control
+
+The Group import allow-list is controlled by the `--group-name` option. You can
+specify multiple filters and a Group will be imported if matches _any_ of the
+supplied filters. Filters can be either literal names, globbed names or Go
+compatible regular expressions. To treat a filter as a regular expression, 
+enclose it in a leading `^` and trailing `$`.
+
+Example filters:
+ - `administrators`: The literal "administrators" group
+ - `site-*`: Any group with the prefix `site-`
+ - `^(?:[^a]|a[^w]|aw[^s]|aws[^\-]).*$`: Any group that does _not_ have the prefix `aws-`
+
+
+<Admonition type="note">
+Ensure that there is no overlap between the groups imported to Teleport and the
+groups you want Okta to maintain control over.
+</Admonition>
+
+<Admonition type="warning">
+Avoid creating an Access List with the same name as a Push Group managed by Okta.
+Teleport will attempt to adopt the group, and may change the group membership.
+Deleting the Teleport Access List and forcing a re-push from Okta should restore 
+access.
+</Admonition>
+
+### User provisioning control
+
+Your Teleport cluster may have a mix of local Teleport users (e.g. a local Admin
+user) and users imported from Okta. By default, Teleport will try to provision
+_all_ Teleport users into Identity Center. You can control which users are 
+provisioned by the Identity Center integration with the `--user-origin` and 
+`--user-filter` arguments. In the example above, the `--user-origin okta` will 
+restrict Teleport to only provisioning users that are synced from Okta, and 
+excluding all local Teleport users.
+
+### AWS Account import control
+
+By default, Teleport will take control of Account Assignments for all AWS Accounts
+managed by Identity Center. You can create an allow-list of AWS Accounts to
+import with the  `--account-name` and `--account-id` install options. 
+
+The `--account-name` filters work like the `--group-name` filters above. The 
+`--account-id` filters specify a literal AWS Account ID.
+
+Teleport will not create or delete Account Assignments on AWS accounts outside
+of its allow-list.
+
+## Step 5: Migrate Account Assignments
+
+We're now at the midway point, and ready to migrate Account Assignments from the
+Okta-managed groups into new Teleport-managed Access Lists. To migrate groups,
+create a new Access List in Teleport (taking care not to use the same name as
+the existing Okta-managed Group) and create the appropriate memberships and 
+Account Assignments.
+
+Account Assignments can be created on an Access List by assigning it the Account
+Assignment roles created by the Identity Center integration, assigning it a 
+custom Teleport role that specifies a specific combination of access, or a
+combination of each.
+
+For more information, see the [Identity Center Integration guide](./aws-iam-identity-center.mdx)
+
+## Step 6: Retire Okta group provisioning
```

Comment:
Should probably also include a section on how to unenroll the Teleport integration side of this as well in case someone decides they don't want to switch yet.